src/Kunstmaan/NodeBundle/Resources/config/services.yml

```diff
@@ -120,3 +120,21 @@ services:
         class: Kunstmaan\NodeBundle\Command\FixTimestampsCommand
         calls:
             - [setContainer, ["@service_container"] ]
+
+    kunstmaan_node.slug.listener:
+        class: Kunstmaan\NodeBundle\EventListener\SlugListener
+        arguments: ["@doctrine.orm.entity_manager","@controller_resolver", "@event_dispatcher"]
+        tags:
+            - { name: kernel.event_listener, event: kernel.controller, method: onKernelController }
+
+    kunstmaan_node.slug.security.listener:
+        class: Kunstmaan\NodeBundle\EventListener\SlugSecurityListener
+        arguments: ["@request_stack" , "@doctrine.orm.entity_manager", "@security.context" , "@kunstmaan_admin.acl.helper" ]
+        tags:
+            - { name: kernel.event_listener, event: kunstmaan_node.slug.security, method: onSlugSecurityEvent }
+
+    kunstmaan_node.render.context.listener:
+        class: Kunstmaan\NodeBundle\EventListener\RenderContextListener
```

Comment:
This class should be a parameter, so users can easily override it with their own implementation.